apps/server/src/modules/provisioning/strategy/oidc/service/schulconnex-group-provisioning.service.ts

```diff
@@ -12,6 +13,7 @@ import { NotFoundLoggableException } from '@shared/common/loggable-exception';
 import { ExternalSource, LegacySchoolDo, Page, UserDO } from '@shared/domain/domainobject';
 import { EntityId } from '@shared/domain/types';
 import { Logger } from '@src/core/logger';
+import { Course } from '../../../../learnroom/domain';
```

Comment:
Please use relative imports when you are importing something from another module